extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -602,15 +602,21 @@ public List<VideoStream> getVideoOnlyStreams() throws ExtractionException {
     }
 
     /**
-     * Try to decrypt url and fallback to given url, because decryption is not
-     * always needed.
+     * Try to decrypt a streaming URL and fallback to the given URL, because decryption may fail if
```

Comment:
Just some grammar fixes:
`      * Try to decrypt a streaming URL and fall back to the given URL, because decryption may fail if`

Reply:
Done, this change will be included in a PR which removes the deprecated code of `YoutubeThrottlingDecrypter`.

I typed this too fast and when I checked the code before pushing it, I was more focused on the code than on the Javadocs and English.